doc/node.1

```diff
@@ -1,368 +1,352 @@
-rules. \fImodule\fR may be either a path to a file, or a node module name.
-
-.TP
-.BR \-\-inspect \fI[=[host:]port]\fR
-Activate inspector on host:port. Default is 127.0.0.1:9229.
-
-V8 Inspector integration allows attaching Chrome DevTools and IDEs to Node.js
-instances for debugging and profiling. It uses the Chrome Debugging Protocol.
-
-.TP
-.BR \-\-inspect-brk \fI[=[host:]port]\fR
-Activate inspector on host:port and break at start of user script.
-
-.TP
-.BR \-\-inspect-port \fI=[host:]port\fR
-Set the host:port to be used when the inspector is activated.
-
-.TP
-.BR \-\-no\-deprecation
+.
+.It Fl r , Fl -require Ar module
+Preload the specified
+.Ar module
+at startup.
+Follows `require()`'s module resolution rules.
+.Ar module
+may be either a path to a file, or a node module name.
+.
+.It Fl -inspect Ar host:port
+Activate inspector on
+.Ar host:port .
+Default is
+.Sy 127.0.0.1:9229 .
+.Pp
+V8 Inspector integration allows attaching Chrome DevTools and IDEs to Node.js instances for debugging and profiling.
+It uses the Chrome Debugging Protocol.
+.
+.It Fl -inspect-brk Ar host:port
+Activate inspector on
+.Ar host:port
+and break at start of user script.
+.
+.It Fl -inspect-port Ar host:port
+Set the
+.Ar host:port
+to be used when the inspector is activated.
+.
+.It Fl -no-deprecation
 Silence deprecation warnings.
-
-.TP
-.BR \-\-trace\-deprecation
+.
+.It Fl -trace-deprecation
 Print stack traces for deprecations.
-
-.TP
-.BR \-\-throw\-deprecation
+.
+.It Fl -throw-deprecation
 Throw errors for deprecations.
-
-.TP
-.BR \-\-pending\-deprecation
+.
+.It Fl -pending-deprecation
 Emit pending deprecation warnings.
-
-.TP
-.BR \-\-no\-warnings
+.
+.It Fl -no-warnings
 Silence all process warnings (including deprecations).
-
-.TP
-.BR \-\-napi\-modules
+.
+.It Fl -napi-modules
 Enable loading native modules compiled with the ABI-stable Node.js API (N-API)
 (experimental).
-
-.TP
-.BR \-\-abort\-on\-uncaught\-exception
+.
+.It Fl -abort-on-uncaught-exception
 Aborting instead of exiting causes a core file to be generated for analysis.
-
-.TP
-.BR \-\-trace\-warnings
+.
+.It Fl -trace-warnings
 Print stack traces for process warnings (including deprecations).
-
-.TP
-.BR \-\-redirect\-warnings=\fIfile\fR
-Write process warnings to the given file instead of printing to stderr.
-
-.TP
-.BR \-\-trace\-sync\-io
-Print a stack trace whenever synchronous I/O is detected after the first turn
-of the event loop.
-
-.TP
-.BR \-\-no\-force\-async\-hooks\-checks
-Disables runtime checks for `async_hooks`. These will still be enabled
-dynamically when `async_hooks` is enabled.
-
-.TP
-.BR \-\-trace\-events\-enabled
-Enables the collection of trace event tracing information.
-
-.TP
-.BR \-\-trace\-event\-categories " " \fIcategories\fR
-A comma separated list of categories that should be traced when trace event
-tracing is enabled using \fB--trace-events-enabled\fR.
-
-.TP
-.BR \-\-zero\-fill\-buffers
+.
+.It Fl -redirect-warnings Ar file
```

Comment:
Can you add the equal signs back on arguments like `−−redirect−warnings=file`? From a quick glance, it looks like the [parser](https://github.com/nodejs/node/blob/d4b605b9903f261c2cadbec3d04822b0de4e14e2/src/node.cc#L3661) is strict about them.

Reply:
What do you mean? Every time I pass `--redirect-warnings=file` (with or without an equals-sign), I get the same error telling me it was given a bad option.

Reply:
I think the option may be actually broken, let me check for another example.

Reply:
Better example: `node --inspect=1234` works while `node --inspect 1234` does not. (Yeah we should really fix our argparser)

Reply:
https://github.com/nodejs/node/issues/18580 for the `--redirect-warnings` issue.

Reply:
> Yeah we should really fix our argparser

Heh. [Use this then](https://github.com/Alhadis/GetOptions). :wink: Good at handling even very complicated option-parsing scenarios.

Reply:
Has to be C++, though :)

Reply:
https://github.com/nodejs/node/issues/18580 resolved. The HTML you generated includes 
U+2122 minus signs which of course throws our parser off as it expects ASCII.

Reply:
> *"The issue appears to not be present in the manpage"*

When you say it's not present, which manpage and manpage version are you specifically referring to? If you're testing them locally, compare the output of these two lines, please:

~~~shell
groff -mandoc -tTascii ./node.1 | less  # Boring old ASCII
groff -mandoc -tTutf8  ./node.1  | less  # UTF-8
~~~

Reply:
Either way, it seems the `=` is necessary, so @Alhadis would you mind adding it back in?

Reply:
Wouldn't it be better to explicitly mention the option requires an equals sign? It isn't immediately clear to the reader that the discrepancy is deliberate.

Reply:
> Wouldn't it be better to explicitly mention the option requires an equals sign?

Not sure I follow. Do you mean adding a "This option requires a equal sign" to each option that does? I'd rather just print it as part of the option title, much more concise.